docs/reference/commandline/cli.md

```diff
@@ -38,6 +38,7 @@ the [installation](../../installation/index.md) instructions for your operating
 For easy reference, the following list of environment variables are supported
 by the `docker` command line:
 
+* `DOCKER_API_VERSION` The API version to use (e.g. `1.19`)
```

Comment:
Per earlier discussions (e.g., "advanced/power user feature", no validation of the format, overriding the API version can lead to unexpected results), should we add a warning here?